sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java

```diff
@@ -106,18 +123,33 @@ public WsConnector wsConnector() {
   }
 
   @Override
-  public OrganizationService organizations() {
+  public OrganizationService organizationsOld() {
+    return organizationsOld;
+  }
+
+  @Override
+  public OrganizationsService organizations() {
     return organizations;
   }
 
+  @Override
+  public org.sonarqube.ws.client.permission.PermissionsService permissionsOld() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "PermissionsService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)